readme.md

```diff
@@ -51,38 +51,19 @@ Default: `false`
 
 Crop to the set height.
 
-##### delay
```

Comment:
Why was this removed? We could use [delay](https://github.com/sindresorhus/delay) to manually delay the capturing after opening the url, no?

Reply:
This was mainly used to wait for elements, we do this automatically now. Although there is probably some option for this in `puppeteer`.

Reply:
It could be used by users as well to make sure an animation has ended. If you know a certain animation takes 2 seconds, someone could use `delay: 2` to make sure he captures after the animation took place.

I never used it to be honest, but just finding a use case :).

Reply:
I would use this. For example, on my website, I load the "latest blog post" widget async, so it's loaded a second after DOMContentLoaded.

Reply:
I think we should switch to a `wait`/`waitFor` option instead that takes a `selector`, a `number` or a `function` and uses this method https://github.com/GoogleChrome/puppeteer/blob/master/docs/api.md#pagewaitforselectororfunctionortimeout-options-args.

Reply:
Yes, good idea, but I also like to keep the delay. Sometimes I just want a screenshot of a random website and ensure it’s fully loaded first.